src/i18n/strings/fi.json

```diff
@@ -3203,5 +3203,6 @@
     "Creating HTML...": "Luodaan HTML:ää...",
     "No virtual room for this room": "Tällä huoneella ei ole virtuaalihuonetta",
     "Switches to this room's virtual room, if it has one": "Vaihtaa tämän huoneen virtuaalihuoneeseen, mikäli huoneella sellainen on",
-    "Removes user with given id from this room": "Poistaa tunnuksen mukaisen käyttäjän tästä huoneesta"
+    "Removes user with given id from this room": "Poistaa tunnuksen mukaisen käyttäjän tästä huoneesta",
+    "All": "Kaikki"
```

Comment:
```suggestion
    "Removes user with given id from this room": "Poistaa tunnuksen mukaisen käyttäjän tästä huoneesta"
```

Reply:
Thanks for the input! I can add some tests if you are happy with how the feature works at the moment.